2016/countdown/index.html

```diff
@@ -53,16 +53,14 @@
 
     <script>
       function update() {
-        var day = new Date(2016,10,08,22),
+        var day = new Date(2016,08,23,22),
             now = new Date(),
             t = day - now,
             seconds = Math.floor((t/1000) % 60),
             minutes = Math.floor((t/1000/60) % 60),
             hours = Math.floor((t/(1000*60*60)) % 24),
-            days = Math.floor((t/(1000*60*60*24)) % 7),
-            weeks = Math.floor(t/(1000*60*60*24*7)),
```

Comment:
I know but there's less than 30 days.

On Fri, Aug 26, 2016 at 12:19 PM, Nat Welch <notifications@github.com>
wrote:

> In 2016/countdown/index.html
> <https://github.com/icco/natwelch.com/pull/10#discussion_r76446154>:
>
> >              now = new Date(),
> >              t = day - now,
> >              seconds = Math.floor((t/1000) % 60),
> >              minutes = Math.floor((t/1000/60) % 60),
> >              hours = Math.floor((t/(1000*60*60)) % 24),
> > -            days = Math.floor((t/(1000*60*60*24)) % 7),
> > -            weeks = Math.floor(t/(1000*60*60*24*7)),
>
> Aww, I love the weeks though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/icco/natwelch.com/pull/10/files/0113b7253d2b0f2018439eba492366af14d1bfc3#r76446154>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAj8sTivy-1cm4ltJMc3usOundfhJeunks5qjxH3gaJpZM4JuFUa>
> .
>